packages/shikiji/src/core/index.ts

```diff
@@ -136,6 +137,34 @@ export async function getHighlighterCore(options: HighlighterCoreOptions) {
     )
   }
 
+  function codeToThemedTokensDualThemes(code: string, options: CodeToThemedTokensDualThemesOptions) {
```

Comment:
Can we reuse this in `codeToHtmlDualThemes`? Inherit the types of options, so we don't even need to have additional tests